### ISSUE TITLE
errors in experimental results 

### ISSUE THREAD

**Issue by hchuphal** [deeplearningTest/deepTest#2]
sudo python2.7 run.py -M mnist_test_model_3_50_leaky_relu
Using TensorFlow backend.
neural_networks/mnist_test_model_3_50_leaky_relu
okx
2020-02-16 20:53:15.830401: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2020-02-16 20:53:15.898393: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x7f84695a61f0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-02-16 20:53:15.898419: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
('Model structure loaded from ', 'neural_networks/mnist_test_model_3_50_leaky_relu')
Validation set filtered for desired class: 0
/Users/hchuphal/Desktop/github/thesis2020/Code/DeepFault_2/utils.py:234: H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
  correct_classifications = group.get('correct_classifications').value
/Users/hchuphal/Desktop/github/thesis2020/Code/DeepFault_2/utils.py:235: H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
  misclassifications = group.get('misclassifications').value
('Classifications loaded from ', 'experiment_results/mnist_test_model_3_50_leaky_relu_0_classifications.h5')
/Users/hchuphal/Desktop/github/thesis2020/Code/DeepFault_2/utils.py:263: H5pyDeprecationWarning: dataset.value has been deprecated. Use dataset[()] instead.
  layer_outs.append(group.get('layer_outs_'+str(i)).value)
('Layer outs loaded from ', 'experiment_results/mnist_test_model_3_50_leaky_relu_0_layer_outs.h5')
(None, 50)
(None, 50)
(None, 50)
('Could not open file: ', 'experiment_results/mnist_test_model_3_50_leaky_relu_C0_tarantula_SN1_suspicious_neurons.h5')

and then no results and nothing in results.log

used default values with the script:
python2.7 run.py -M mnist_test_model_3_50_leaky_relu

are there any issues in file creation on macbook or issues with argumets ie.. running with default values?
